pkgs/applications/misc/tvbrowser/test.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, xvfb-run
+, tvbrowser
+, runCommand
+, writeShellApplication
+, xorg
+}:
+
+let
+  testScript = writeShellApplication {
+    name = "tvbrowser-test-script";
+    runtimeInputs = [ xorg.xwininfo tvbrowser ];
+    text = ''
+      function find_tvbrowser_windows {
+          for window_name in java tvbrowser-TVBrowser 'Setup assistant' ; do
+              grep -q "$window_name" "$1"  ||  return 1
+          done
+      }
```

Comment:
We use 2 spaces unless the file is a shellscript, so not for shellscripts in nix files.